readthedocs/profiles/tests/test_views.py

```diff
@@ -358,16 +365,6 @@ def test_migration_page_step_connect_done(self, request):
         assert response.status_code == 200
         context = response.context
         assert context["installation_target_groups"] == [
-            InstallationTargetGroup(
```

Comment:
With the new repository, the order changed. We can't use a set here because the dataclasse isn't hasheable.